rules/sources/personal_identification.yaml

```diff
@@ -38,14 +38,15 @@ sources:
       - "(?i).*((?:last|sur(?!geon))[^\\s/(;)#|,=!>]{0,5}name)"
     tags:
       law: GDPR
-  
+
   - id: Data.Sensitive.PersonalIdentification.Age
     name: Age
     category: Personal Identification
     isSensitive: False
     sensitivity: low
     patterns:
-      - "(?i)age|is[_-]{0,1}(minor|major|adult|child)|(.*(age[^\\s/(;)#|,=!>]{0,10}(?:years|yrs)|(?:user|person|customer)[^\\s/(;)#|,=!>]{0,5}(?<!im|p|langu|us|aver)age))"
+      - "(?i)^(user|person|customer|under|years|yrs|human)?(age)(human|years|yrs|user|person|customer)?$"
```

Comment:
@gaurav-gogia test out this, this currently matches, `agehuman, ageuser` etc which doesn't seem appropriate, variables like `ageOfUser` is missed out